pkgs/development/libraries/gpgme/default.nix

```diff
@@ -18,11 +26,21 @@ stdenv.mkDerivation rec {
     [ libgpgerror glib libassuan pth ]
     ++ lib.optional (qtbase != null) qtbase;
 
-  nativeBuildInputs = [ pkgconfig gnupg ];
+  nativeBuildInputs = [ file pkgconfig gnupg autoreconfHook git texinfo5 ]
+  ++ lib.optionals withPython [ python swig2 which ];
+
+  preConfigure = ''
+    ./autogen.sh
```

Comment:
is this needed with the `autoreconfHook`?

Reply:
not anymore indeed.